### ISSUE TITLE
XML Literal Hot Reload Issue

### ISSUE THREAD

**Issue by CoolCoderSuper** [dotnet/roslyn#75231]
**Version Used**: 17.12.0 Preview 2.0

**Steps to Reproduce**:

1. Edit text content in XML literal element or attribute and no changes will be detected.

If you add or remove an element or attributes hot reload works as expected.

**Expected Behavior**:
Hot reload would procede as normal.
**Actual Behavior**:
No changes are detected.

**Comment by CoolCoderSuper** [comment]
Did some more test and it looks like if you add an additional text node for example
```xml
<div>
cdscdsdcs
<br/>
</div>
```
to
```xml
<div>
cdscdsdcs
<br/>
hhhhhhhhhhhhhh
</div>
```
Hot reload will work.

If you change the type of the tag hot reload will fail.

It seems that changes to existing nodes fail but additions and removals are detected properly.

**Comment by CoolCoderSuper** [comment]
I attempted to debug it my self but I could figure out how to debug the remote services.
Is there a guide for this?

**Comment by CoolCoderSuper** [comment]
I'm assuming the issue would be in VisualBasicEditAndContinueAnalyzer.
If this is the case are most edit "automatically" dealt with in AbstractEditAndContinueAnalyzer?
I assume that XML literals have already been lowered to constructor calls at this point if not how or when does that happen?
